src/config.rs

```diff
@@ -145,6 +148,8 @@ impl Default for Server {
             version_id_cache_ttl: Duration::from_secs(
                 env_optional("VERSION_ID_CACHE_TTL").unwrap_or(DEFAULT_VERSION_ID_CACHE_TTL),
             ),
+            max_new_versions_daily: env_optional("MAX_NEW_VERSIONS_DAILY")
+                .unwrap_or(DEFAULT_MAX_NEW_VERSIONS_DAILY),
```

Comment:
I think I would prefer it if we kept `max_new_versions_daily` an `Option` without the default value. That way we can easily enable/disable this when we deploy the code.

```suggestion
            max_new_versions_daily: env_optional("MAX_NEW_VERSIONS_DAILY"),
```

Reply:
Yes, it's not a problem